### DISCUSSION TITLE
It would be nice to be able to hide dotfiles/config files by default on repo code view

### DISCUSSION THREAD

**Discussion by NatoNathan** [community/community#47892] [Repositories]
### Select Topic Area

Product Feedback

### Body

On some projects/repos that have a lot of dotfiles or config files in the repos root it would be nice to be able to hide them in the code view by default, but keep them easily accessible.

**Comment by AsafMeizner** [comment]
It depends on what code editor or tool you are using to view your code. For example, if you are using Visual Studio Code, you can configure patterns to hide files and folders from the explorer and searches. If you want to store your dotfiles in a Git bare repository, you can use an alias to interact with your configuration repository. If you want to hide dotfiles from Windows Explorer, you can use a system call API injection.

If you are using GitHub Atom editor, you can go to Edit > Preferences > Packages and check the “Hide Ignored Names” option. [Then you can enter .* in the Ignored Names box to hide all dotfiles. [If you are using Git bare repository, you can use the same method as I mentioned before. If you want to ignore dotfiles or config files from Git, you can use a .gitignore file and specify the patterns of files you want to ignore.

other than that the only thing i can ofer is for you to put all those files in a folder and manage everything difrently or use a git ignore but i dont think thats what you meant

**Reply by NatoNathan** [reply]
I mean in the github web interface, I know I can hide them locally (tbf i want to see the dotfiles of local repos anyway) but when browsing repos on github I don't always want to see all the dotfiles/ config files. I may only care about the source files and/or doc files.

**Reply by AsafMeizner** [reply]
There is no way yet to do that, all you could do is email them (this is not a place for requests) so just sign this as finished, sorry I couldn't help

**Reply by NatoNathan** [reply]
I'm sorry but this whole discussion forum is for Product Feedback, questions, and conversations about GitHub Repositories. Feature requests would fall under Product Feedback, no? It does on the other product topics on this forum.

**Reply by AsafMeizner** [reply]
its feedback incase something is wrong or broken, not for requests from my understanding

**Reply by NatoNathan** [reply]
I'm not sure where you got that impression, from my previous interactions on this forum the github team welcome feature requests.

**Reply by AsafMeizner** [reply]
then I'm sorry, I was wrong
